buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateEapBuildNumber.kt

```diff
@@ -14,18 +15,18 @@ open class UpdateEapBuildNumber : DefaultTask() {
     val htmlContent = Jsoup.connect(intellijSnapshotsUrl).get()
     val links =
       htmlContent.select(
-        "a[href^=${intellijSnapshotsUrl}com/jetbrains/intellij/idea/ideaIC/][href$=\"-EAP-SNAPSHOT.pom\"]"
+        "a[href^=${intellijSnapshotsUrl}com/jetbrains/intellij/idea/ideaIC/][href$=-EAP-SNAPSHOT.pom]"
       )
 
     for (link in links) {
       val attr = link.attr("href")
-      val regex = Regex("(?<=ideaIC-)\\d+\\.\\d+\\.\\d+(?=-EAP-SNAPSHOT.pom)")
+      val regex = Regex("(?<=ideaIC-)(\\d+\\.)+\\d+(?=-EAP-SNAPSHOT.pom)")
```

Comment:
Oh, to capture `222.1234` format rather than `222.1234.56`? are there such occurrences?

Reply:
Looks like there are NONE, will revert it.